src/components/pickers/PeoplePicker/PeoplePicker.tsx

```diff
@@ -15,47 +15,7 @@ export interface IPeoplePickerProps extends IBasePickerProps<IPersonaProps> {
 export class BasePeoplePicker extends BasePicker<IPersonaProps, IPeoplePickerProps> {
 }
 
-export class MemberListPeoplePicker extends BasePeoplePicker {
-
-  public render() {
-    let { displayValue } = this.state;
-
-    return (
-      <div>
-        <div ref={ this._resolveRef('_root') } className='ms-BasePicker' onKeyDown={ this._onKeyDown }>
-          <SelectionZone selection={ this._selection }
-            selectionMode={SelectionMode.multiple}>
-            <div className='ms-BasePicker-text'>
-              <input ref={ this._resolveRef('_input') }
-                onFocus={ this._onInputFocus }
-                onChange={ this._onInputChange }
-                value={ displayValue }
-                className='ms-BasePicker-input'
-                />
-            </div>
-          </SelectionZone>
-        </div>
-        { this._renderSuggestions() }
-        <FocusZone ref={ this._resolveRef('_focusZone') }>
-          { this._renderItems() }
-        </FocusZone>
-
-      </div>
-    );
-  }
-
-  protected _onBackspace(ev: React.KeyboardEvent) {
-    let { value } = this.state;
-    if (ev.target === this._input) {
-      if (value && this._input.selectionStart !== this._input.selectionEnd) {
-        this._updateValue(value.substring(0, this._input.selectionStart));
-        // Since this effectively deletes a letter from the string we need to preventDefault so that
-        // the backspace doesn't try to delete a letter that's already been deleted. If a letter is deleted
-        // it can trigger the onChange event again which can have unintended consequences.
-        ev.preventDefault();
-      }
-    }
-  }
+export class MemberListPeoplePicker extends BasePickerListBelow<IPersonaProps, IPeoplePickerProps> {
```

Comment:
`BasePickerListBelow` is a really weird name...

Would it make more sense to create a PeoplePicker with an enum of variants? does the logic really differ that greatly?